tests/datatypes/test_bytes.py

```diff
@@ -28,6 +30,12 @@ def test_capitalize(self):
             print(b'2015638687'.capitalize())
         """)
 
+    @unittest.skip("Move this to test_capitalize upon resolution of #530")
```

Comment:
This should be expectedFailure, rather than skip. skip is for when a test may/may not run due to some other condition (e.g., whether a setting has been turned on) - in this case, the "setting" is a string, which is always true, so the test won't ever be executed.